shared/resources/styles/components/exercise/statics.less

```diff
@@ -0,0 +1,75 @@
+&-two-step-intro,
+&-personalized-intro,
+&-spaced-practice-intro {
+
+  h1 {
+    display: flex;
+    align-items: center;
+
+    &::before {
+      padding-right: 1rem;
+    }
+  }
+
+  &.card-body { // overide normal card-body styles
+    padding: 60px 140px;
+  }
+
+  .heading {
+    text-transform: uppercase;
+    display: none;
+  }
+
```

Comment:
eh good point, it was displayed before but the text was being uppercased in the content itself.  might as well take this out since the design doesn't have this anymore